index.bs

```diff
@@ -1388,6 +1389,12 @@ that are returned to the caller when a new credential is created, or a new asser
         {{CredentialsContainer/create()}}, this attribute's value will be an {{AuthenticatorAttestationResponse}}, otherwise,
         the {{PublicKeyCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
         will be an {{AuthenticatorAssertionResponse}}.
+    
+    :   <dfn>authenticatorAttachment</dfn>
+    ::  This attribute contains the authenticatorAttachment used to communicate to the [=authenticator=], or `null` if 
+        the user agent does not have any attachment information. This attribute should be used by RPs in conjecture 
+        with isUVPAA to prompt user to register platform authenticator if it is availible and user is currently being authenticated 
```

Comment:
```suggestion
        with `isUserVerifyingPlatformAuthenticatorAvailable()` to prompt user to register a platform authenticator if it is available and the user is currently being authenticated 
```

Reply:
The reference to isUserVerifyingPlatformAuthenticatorAvailable should be a spec link